README.md

```diff
@@ -1 +1,13 @@
-# building-a-checks-api-ci-server
\ No newline at end of file
+This is the sample project he built by following the "[Building Your First GitHub App](https://developer.github.com/apps/building-your-first-github-app)" Quickstart guide on developer.github.com.
+
+It consists of two different servers: `server.rb` (boilerplate) and `advanced_server.rb` (completed project).
```

Comment:
What are your thoughts on progressing the evolution of `server.rb` through branches vs in different files on the same branch?

Reply:
I'm still considering this because I love how elegant it would be to show the progress. But, we're talking about breaking out the template code (server.rb) into its own repository because it will serve as a template for future GitHub App guides, and we want to be as DRY as possible. I am keeping this in mind though until we settle on a solution.